app/controllers/JobController.scala

```diff
@@ -106,81 +103,26 @@ class JobController @Inject() (override val config:Configuration, override val c
-    * @return
-    */
-  def updateStatus(jobId:String) = Action.async(circe.json(2048)) { request=>
-    jobModelDAO.jobForId(jobId).flatMap({
-      case None=>
-        Future(NotFound(GenericErrorResponse("not_found",s"no job found for id $jobId").asJson))
-      case Some(Left(err))=>
-        Future(InternalServerError(GenericErrorResponse("error", err.toString).asJson))
-      case Some(Right(jobDesc))=>
-        JobReport.getResult(request.body) match {
-          case None=>
-            Future(BadRequest(GenericErrorResponse("error","Could not decode any job report from input").asJson))
-          case Some(result)=>result match {
-            case report:JobReportError=>
-              val finalReport = report.decodeLog match {
-                case Success(decodedReport)=>
-                  logger.error(s"Outboard process indicated job failure (successfully decoded): $decodedReport")
-                  decodedReport
-                case Failure(err)=>
-                  logger.warn(s"Could not decode report: ", err)
-                  report
-              }
-              val updatedJd = jobDesc.copy(completedAt = Some(ZonedDateTime.now),jobStatus = JobStatus.ST_ERROR, log=Some(finalReport.log))
-              jobModelDAO.putJob(updatedJd)
-                .map(result=>Ok(GenericErrorResponse("ok","received report").asJson))
-
-            case report:JobReportSuccess=>
-              implicit val esClient = esClientManager.getClient()
-              implicit val s3Client = s3ClientManager.getClient(awsProfile)
-              implicit val ddbClient = ddbClientManager.getNewAlpakkaDynamoClient(awsProfile)
-
-              logger.info(s"Outboard process indicated job success: $report")
-
-              val proxyUpdateFuture = JobControllerHelper.thumbnailJobOriginalMedia(jobDesc).flatMap({
-                case Left(err)=>Future(Left(err))
-                case Right(archiveEntry)=>JobControllerHelper.updateProxyRef(report, archiveEntry, proxyLocationDAO)
-              })
-
-              proxyUpdateFuture.flatMap({
-                case Left(err)=>
-                  logger.error(s"Could not update proxy: $err")
-                  val updatedJd = jobDesc.copy(completedAt = Some(ZonedDateTime.now),log=Some(s"Could not update proxy: $err"), jobStatus = JobStatus.ST_ERROR)
-                  jobModelDAO.putJob(updatedJd)
-                    .map(result=>Ok(GenericErrorResponse("ok",s"received report but could not update proxy: $err").asJson))
-                case Right(msg)=>
-                  val updatedJd = jobDesc.copy(completedAt = Some(ZonedDateTime.now),jobStatus = JobStatus.ST_SUCCESS)
-                  jobModelDAO.putJob(updatedJd)
-                    .map(result=>Ok(GenericErrorResponse("ok","received report").asJson))
-              })
-          }
-        }
-    })
-  }
 
   def refreshTranscodeInfo(jobId:String) = APIAuthAction.async { request=>
     implicit val timeout:akka.util.Timeout = 30 seconds
 
```

Comment:
this needs to be re-implemented via the Proxy Framework, will be done in a future release